api/conferenceHall.json

```diff
@@ -951,7 +951,7 @@
         "_seconds": 1697189511,
         "_nanoseconds": 190000000
       },
-      "slidesLinks": []
+      "slidesLinks": ["https://touraine.tech/slides/yannbertrand-2024-grosse-fonctionnalite-petites-livraisons.pdf"]
```

Comment:
remove https://touraine.tech/ from the link.
This will break when the 2024 edition will be archived !
The link should be only : 
"slides/yannbertrand-2024-grosse-fonctionnalite-petites-livraisons.pdf"

Reply:
Done!